spacy/errors.py

```diff
@@ -934,6 +934,7 @@ class Errors(metaclass=ErrorsWithCodes):
     E1041 = ("Expected a string, Doc, or bytes as input, but got: {type}")
     E1042 = ("Function was called with `{arg1}`={arg1_values} and "
              "`{arg2}`={arg2_values} but these arguments are conflicting.")
+    E1043 = ("Expected None or a value in range [{range_start}, {range_end}] for {metric}, but got {value}.")
```

Comment:
So in general this would be a good idea to keep the number of error codes down, but this is only used with the EntityLinker threshold, right? In that case I think it should be fine to just change `{metric}` to `"entity linker threshold"`. This does have the advantage that the error code indicates a specific failure and not a generic one.